.github/workflows/kata-deploy-test.yaml

```diff
@@ -5,13 +5,36 @@ on:
 name: test-kata-deploy
 
 jobs:
-  build-asset:
+  check-comment-and-membership:
+    runs-on: ubuntu-latest
     if: |
       github.event.issue.pull_request
       && github.event_name == 'issue_comment'
       && github.event.action == 'created'
       && startsWith(github.event.comment.body, '/test_kata_deploy')
+    steps:
+      - name: Check membership
+        uses: JamesSingleton/is-organization-member@1.0.1
```

Comment:
Sorry about the distrust for third party actions, but I have already reviewed the action and it looks good to me but as it interacts with the github token. I prefer to fork that action on our org and upgrade the branch if needed in the future.

Reply:
@jcvenegas, I forked the repo onto kata-containers org and now we're using it from there.
This makes sense, mainly considering how non-restrictive is our actions configurations, which is something we should re-evaluate at some point soon.

Reply:
Thanks @jcvenegas. Talking in general terms but if we've reviewed the code, in theory we could just specify the git commit for the version we want to use since that cannot change (without us noticing), whereas the numeric (tag) version _could_ change. But this coment applies to all GitHub actions versions so isn't specific to this PR.